README.md

```diff
@@ -117,6 +117,49 @@ kubectl create -f https://raw.githubusercontent.com/logdna/logdna-agent/master/l
 
 This automatically installs a logdna-agent pod into each node in your cluster and ships stdout/stderr from all containers, both application logs and node logs. Note: By default, the agent pod will collect logs from all namespaces on each node, including `kube-system`. View your logs at https://app.logdna.com. See [YAML file](https://raw.githubusercontent.com/logdna/logdna-agent/master/logdna-agent-ds.yaml) for additional options such as `LOGDNA_TAGS`.
 
+### Upgrading to LogDNA Agent 2.0 for Kubernetes
+
+We've recently made the LogDNA Agent 2.0 publicly available for Kubernetes users. We'll be rolling this out to existing users as well as other platforms and operating systems over the next few weeks, but if you'd like to upgrade your existing Kubernetes agent and have an existing YAML file, you can simple change
+
+```yaml 
+image: logdna/logdna-agent:latest
+``` 
+to 
+```yaml 
+image: logdna/logdna-agent-v2:stable
+```
+
+Then re-apply your YAML and restart the agent. To confirm please run `kubectl get ds logdna-agent -o yaml | grep "image: logdna/"` and if you see `image: logdna/logdna-agent-v2:latest` then you are good to go.
```

Comment:
image: logdna/logdna-agent-v2:latest should be image: logdna/logdna-agent-v2:stable